packages/react-magma-dom/src/components/TreeView/TreeItem.test.js

```diff
@@ -75,6 +74,37 @@ describe('TreeItem', () => {
     });
   });
 
+  describe('additional content styles', () => {
```

Comment:
we should add tests for confirming that the interactive elements get called when clicked or space/enter are pressed